app/v6/foreignPropertyBsas/retrieve/RetrieveForeignPropertyBsasController.scala

```diff
@@ -37,6 +38,11 @@ class RetrieveForeignPropertyBsasController @Inject() (
     extends AuthorisedController(cc)
     with Logging {
 
+  val endpointName = "retrieve-foreign-property-bsas"
+
+  lazy protected val supportingAgentsAccessControlEnabled: Boolean =
+    BsasFeatureSwitches().supportingAgentsAccessControlEnabled
```

Comment:
If this is the same in every controller, why is this not in the AuthorisedController (I suppose it comes down to why does it need to use a BSAS-specific feature switches class?)?